src/App.js

```diff
@@ -22,12 +69,6 @@ export default function App() {
     setBoard([]);
     setSDeck(sideDeck);
     setSideBoard([]);
-  }
-
-  // createCard maps the board with all drawn cards
-
-  const createCard = (card) => {
```

Comment:
This isn't wrong to do a mapping this way, but it's easier to follow if you just have the map callback function where the `.map` function is called. (Line 110 in this case)